stylesheets/modules/b2b-account/account-hierarchy.less

```diff
@@ -0,0 +1,190 @@
+.account-hierarchy {
+    margin-bottom: 50px;
+
+    ul.account-hierarchy-menu {
+        margin-bottom: 20px;
+        padding-left: 0px !important;
+
+        > li {
+            display: inline-block;
+            font-size: 16px;
+            cursor: pointer;
+        }
+
+        > li::after {
+            content: "|";
+            margin: 10px;
+        }
+
+        > li:last-child:after {
+            content: "";
+        }
+    }
+    //Tree Styles
+    ul.tree, ul.tree ul {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+    }
+
+    ul.tree ul {
+        margin-left: 16.5px;
+        margin-top: 5px;
+    }
+
+    ul.tree li {
+        margin: 0;
+        padding: 5px 7px;
+        line-height: 32px;
+        border-left: 2px solid rgb(100, 100, 100);
+    }
+
+    ul.tree li:last-child {
+        border-left: none;
+    }
+
+    ul.tree li:before {
+        position: relative;
+        top: -5px;
+        height: 1.4em;
+        width: 45px;
+        color: white;
+        border-bottom: 2px solid rgb(100, 100, 100);
+        content: "";
+        display: inline-block;
+        left: -8px;
+        z-index: -10;
+    }
+
+    ul.tree li:last-child:before {
+        border-left: 2px solid rgb(100, 100, 100);
+        left: -7px;
+    }
+
+    .caret {
+        cursor: pointer;
+        -webkit-user-select: none;
+        /* Safari 3.1+ */
+        -moz-user-select: none;
+        /* Firefox 2+ */
+        -ms-user-select: none;
+        /* IE 10+ */
+        user-select: none;
+    }
+
+    .caret::before {
+        content: '';
+        background: url('../resources/images/plus-solid.svg');
+        background-color: #fff;
+        color: black;
+        display: inline-block;
+        margin-right: 3px;
+        height: 16px;
+        width: 16px;
+        left: -8px;
+        position: relative;
+        top: 1.5px;
+    }
+
+    .caret::after {
+        border-left: 2px solid rgb(100, 100, 100);
+    }
+
+    .caret-user {
```

Comment:
Caret user isn't used anymore. Can remove it as well as the unused image files.

Reply:
Done